sam_view.c

```diff
@@ -465,6 +471,12 @@ int main_samview(int argc, char *argv[])
             break;
         case 'M': settings.multi_region = 1; break;
         case 1: no_pg = 1; break;
+        case 'z':
+            if (!(settings.filter = hts_filter_init(optarg))) {
+                fprintf(stderr, "main_samview: Failed to parse filter\n");
```

Comment:
`print_error()`

Reply:
Heh, I just copied the local style from about 10 lines above (-x), but can change it.

Reply:
16 `/fprintf.*main_samview/` vs 29 `/print_error/` in _sam_view.c_ — gotta keep that ratio improving one line at a time :smile: